### DISCUSSION TITLE
[Unknown Mod][Adventure Map] Display the artifact name in the artifact's right-click info window

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#7841] [Ideas]
Display the artifact name in the artifact's right-click info window.

In fheroes2:

![Artifact info 001](https://github.com/ihhub/fheroes2/assets/43583994/e3a2f5d1-8fd9-41b3-887f-57b7be4c606e)

This is an option.

**Comment by Branikolog** [comment]
Hi, @LeHerosInconnu 
I personally don't like this idea, as in Heroes 2 there are cursed artifacts, which player could pick up and then be surprised by his luck.
Sometimes map maker hide those artifacts behind some terrain objects etc. Showing the name will make less fun with this game aspect. :)

**Reply by LeHerosInconnu** [reply]
Hello @Branikolog,

The fact that artifacts can be hidden by decors only means that the player will make a save before capturing the artifact, and reload if it doesn't suit him.
This runs counter to the player's enjoyment of the game.

**Reply by Granfalegion** [reply]
You could say the same about any negative outcome in the game.  Players reload all the time.

Encountering negatives, especially surprising ones, is not the same as not enjoying the game.

**Reply by LeHerosInconnu** [reply]
Hello @Granfalegion,

In the specific case of artifacts hidden by the decor, this is due to the 2D representation of the adventure map used in the game.
This can easily be improved, saving the player from unwanted and annoying saves and reloads.

**Reply by Granfalegion** [reply]
I understand the issue.  But I am concurring with @Branikolog that this is not necessarily a bad thing.  Your argument that a player might need to reload if a bad outcome arises does not strike me as being something that is disqualifying or even unwanted.  As has been pointed out, there are some map designs that count on this particular element.

**Reply by LeHerosInconnu** [reply]
@Granfalegion.

So, some people will have fun with the hidden artifacts, while others won't.
In what percentage?
Should an option be offered in this case?
